reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -358,6 +395,12 @@ def get_item_based_topk(self, items, top_k=10, sort_top_k=False):
         # calculate raw scores with a matrix multiplication
         test_scores = pseudo_affinity.dot(self.item_similarity)
 
+        if normalize:
```

Comment:
I would suggest we use the rescaling only for obtaining ratings from the original SAR scores. The recommendations of items here and in `recommend_k_items()` would be based on the original matrices.

Reply:
yeah, this makes sense, i'll take it out except for explicit use during scoring